UPGRADE-4.1.md

```diff
@@ -0,0 +1,18 @@
+UPGRADE FROM 4.0 to 4.1
+=======================
+
+Security
+--------
+
+ * the `ContextListener::setLogoutOnUserChange()` method is deprecated and will be removed in 5.0
```

Comment:
Missing cap and dot (same below) and in other files.

Reply:
fixed